embulk-standards/src/main/java/org/embulk/standards/LocalFileInputPlugin.java

```diff
@@ -103,17 +105,44 @@ public void cleanup(TaskSource taskSource,
             int taskCount,
             List<TaskReport> successTaskReports) {}
 
+    String escapeGlobPattern(String filenamePrefix) {
```

Comment:
Can you :
* Make this method `private`
* Make it return `PathMatcher` directly
* Rename it to `buildPathMatcherForFileNamePrefix`
* Move it at the bottom of this class

Reply:
I agree.

That comment message has a typo?

```
s/case-insensitively/case-sensitively/
```